packages/env/lib/config/test/config.js

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable jest/no-try-expect, jest/no-conditional-expect */
+/* eslint-disable jest/no-conditional-expect */
```

Comment:
The `jest/no-try-expect` rule was removed with v25: https://github.com/jest-community/eslint-plugin-jest/releases/tag/v25.0.0